static/search.js

```diff
@@ -0,0 +1,98 @@
+"Use strict";
+const address = 'https://www.googleapis.com/books/v1/volumes?q=';
+const key = '&key=AIzaSyAqMcQj6rxs6dXYzTytATx9lz558CHvKCc';
+let query = async (searchterms) => {
+    let searchBy = document.querySelector('[name=searchBy]');
+    let parameters = searchBy.options[searchBy.selectedIndex].value;
+    let request = address + parameters + searchterms + key;
+    let response = await (fetch(request)
+        .then(res => {
+            return res.json()
+        })
+        .catch(err => {
+            console.log('Error: ', err);
+        })
+    )
+    console.log(response);
+    let results = response.items;
+    populate(results);
+}
+
+let getBookData = (volume) => {
+    let thumbnailSrc;
+    let defaultImg = 'http://nobacks.com/wp-content/uploads/2014/11/Book-2.png';
+    thumbnailSrc = volume["imageLinks"].smallThumbnail
+        // thumbnailSrc = JSON.stringify(volume.imageLinks.smallThumbnail);
+        
+    console.log(thumbnailSrc);
+    if (thumbnailSrc == false) {
+        thumbnailSrc = defaultImg;
+    }
+    let bookData = {
+        title: volume.title,
+        author: volume.authors,
+        thumbnail: thumbnailSrc,
+        publishedDate: volume.publishedDate
+    };
+    let identifiers = volume.industryIdentifiers;
+    console.log(identifiers);
+    for (var i=0; i<identifiers.length; i++) {
+        if (identifiers[i].type["ISBN_10"]) {
+            let asin = identifiers[i].type["ISBN_10"];
+            console.log(asin);
+        }
+        if (identifiers[i].type["ISBN_10"]) {
```

Comment:
Is this supposed to say .type["ISBN_13"]?

Reply:
they provide multiple types of identifiers and there are 2 types of ISBN's - 10 digit and 13 digit. So this is just determining which type(s) are present. We could ignore the 10 digit ISBN though, and just save the ISBN 13 for our purposes.